README.md

```diff
@@ -5,35 +5,49 @@ The digital publishing platform is a collection of services which provide
 content management and publishing functionality for ONS content, and is
 responsible for the public ONS website.
 
-### Guides
-
-* [Getting started](GETTING_STARTED.md)
-* [Development standards](DEV_STANDARDS.md)
-* [Definition of Ready](DEFINITION_OF_READY.md)
-* [Definition of Done](DEFINITION_OF_DONE.md)
-* [Release process](RELEASES.md)
+Guides and general info
+-----------------------
+
+* [Getting started](guides/GETTING_STARTED.md)
+* [Release process](guides/RELEASES.md)
+* [Contributing guidelines](guides/CONTRIBUTING.md)
+* [GPG commit signing](guides/GPG.md)
+* [Port numbering](guides/PORTS.md)
+* [Technologies](guides/TECHNOLOGIES.md)
+* [Tagging a release](guides/TAGS.md)
+
+Standards and specifications
+----------------------------
+
+* [Development standards](standards/DEV_STANDARDS.md)
+* [Definition of Ready](standards/DEFINITION_OF_READY.md)
+* [Definition of Done](standards/DEFINITION_OF_DONE.md)
 * [Digital Publishing principles](https://github.com/ONSdigital/dp-principles)
-* [Contributing guidelines](CONTRIBUTING.md)
-* [GPG commit signing](GPG.md)
-* [Port numbering](PORTS.md)
-* [Technologies](TECHNOLOGIES.md)
-* ONS CDN
-  * [ONS assets](https://github.com/ONSdigital/cdn.ons.gov.uk-assets)
-  * [Vendored dependencies](https://github.com/ONSdigital/cdn.ons.gov.uk-vendor)
+* [API standards](standards/API_STANDARDS.md)
+* [Logging standards](standards/LOGGING_STANDARDS.md)
+* [Health check specification](standards/HEALTH_CHECK_SPECIFICATION.md)
+
+Dev tooling
+-----------
+
+**[dp tool](cmd/README.md)**
+
+A command line utility to help make working in Digital Publishing simpler.
```

Comment:
I think that `dp` exposes some things that should not be public, so should be moved into dp-ci or similar? (If we're having a big tidy, here)

Reply:
I think the `dp` tool and the `paasbox` and `websysd` stuff should probably be split out into a new repo.  Whether these should be private is a discussion we can have at that time.